client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -196,10 +210,13 @@ export default connect( state => {
 		purchasesError: getPurchasesError( state ),
 		importUrl,
 		exportUrl,
+		cloneUrl,
 		showChangeAddress: ! isJetpack && ! isVip,
+		showClone: 'active' === rewindState.state,
```

Comment:
this means that the knowledge of being able to clone a site may be fickle as the rewind state moves through different credential states and server issues.

if instead we reacted to the state in different ways then I think it would help discoverability and minimize confusion.

for this immediate task that could be as simple as showing whether cloning is enabled/disabled - we'll talk more in person